tests/unit_tests/shared/share/test_shell.py

```diff
@@ -228,35 +228,44 @@ def test_move_pathfile_into_folder_file_exists(shell):
 
 
 @pytest.mark.usefixtures("use_tmpdir")
-def test_delete_file(shell):
+def test_that_delete_file_cannot_delete_directories(shell):
     shell.mkdir("pathx")
     assert b"not a regular file" in shell.delete_file("pathx").stderr
 
-    # deleteFile which does not exist - is silently ignored
-    shell.delete_file("does/not/exist")
 
-    with open("file", "w") as f:
-        f.write("hei")
+@pytest.mark.usefixtures("use_tmpdir")
+def test_that_delete_file_ignores_non_existing_files(shell):
+    assert b"ignored" in shell.delete_file("does/not/exist").stderr
+
+
+@pytest.mark.usefixtures("use_tmpdir")
+def test_that_delete_file_deletes_broken_symlinks(shell):
+    Path("file").write_text("hei", encoding="utf-8")
     shell.symlink("file", "link")
-    assert os.path.islink("link")
+    assert Path("link").is_symlink()
 
     shell.delete_file("file")
-    assert not os.path.isfile("file")
-    assert os.path.islink("link")
+    assert not Path("file").exists()
+    assert Path("link").is_symlink() and not Path("link").exists()
     shell.delete_file("link")
-    assert not os.path.islink("link")
+    assert not Path("link").is_symlink() and not Path("link").exists()
 
 
 @pytest.mark.usefixtures("use_tmpdir")
-def test_delete_directory(shell):
-    # deleteDriecteory which does not exist - is silently ignored
-    shell.delete_directory("does/not/exist")
+def test_that_delete_non_existing_directory_is_silently_ignored(shell):
```

Comment:
I would remove `that` from the test function names, which applies for other tests too.

Reply:
I added `_that_` based on the discussion and recommendations in the scout channel on slack around dec 5

Reply:
Right, when so it can go as is :)